docs/static/config-details.asciidoc

```diff
@@ -87,6 +90,35 @@ Keep the overall memory requirements in mind when you allocate memory.
 By default, a JVM's off-heap direct memory limit is the same as the heap size. Check out <<plugins-inputs-beats-memory,beats input memory usage>>.
 Consider setting `-XX:MaxDirectMemorySize` to half of the heap size.
```

Comment:
This entry should be moved down, and enhanced with some reasoning, since the new content introduces how to set it, so guidance on the values should come afterwards.

Reply:
Updated by commit d692dc1466c94a2d449108b50dc9e67d1b3483a4